gempy/assets/geographic.py

```diff
@@ -0,0 +1,562 @@
+""" Method to handle geographic information
+
+General set of methods to handle geographic inforamtion and
+additional data sets (e.g. GoogleEarth .kml files, GeoTiffs, etc.)
+
+2020 Florian Wellmann
+"""
+
+try:
+    from osgeo import ogr, osr
+    import pyproj
+    import gdal
```

Comment:
have you investigated rasterio yet?

for my own work I tend to use the rasterio/shapely/fiona/geopandas stack of packages as they are all made to work together and implement a lot of the functionality in this file.

For example in geopandas, you can read a shapefile into a GeoDataFrame (basically a pandas dataframe) and you can reproject to a different crs using `.to_crs('4326')` for example. see https://geopandas.org/projections.html

Reply:
Thanks for the review and the very helpful advice! We are currently actually experimenting in a separate repository with a couple of these packages (didn't want to mess around too much with gempy until we see better what to use): 

https://github.com/cgre-aachen/gemgis/tree/development_gemgis_alex_juestel

Geopandas looks promising, but we also didn't want to completely change the pandas data frames in gempy to geopandas due to the additional dependencies. So our current plan is to have a very light-weight addition to the gempy core, and then to add functionality in the data preprocessing part (where using geopandas would make sense).

A bit of a difficult bridge...